.gitignore

```diff
@@ -0,0 +1,4 @@
+*.pyc
+Config.ini
```

Comment:
How come this file has stuff in it now?

Reply:
Hmm I added it (in fact i always had it on my side but never submitted), do you want me to keep it locally only ?

Reply:
Ah my mistake I thought it was the .gitkeep file.